buildenv

```diff
@@ -1,84 +1,192 @@
 export ZOPEN_STABLE_URL="https://github.com/boostorg/boost.git"
-#export ZOPEN_STABLE_TAG="boost-1.82.0"
-export ZOPEN_DEV_BRANCH="boost-1.82.0"
 export ZOPEN_STABLE_DEPS="cmake make zoslib which coreutils comp_clang sed jq"
 export ZOPEN_DEV_URL="https://github.com/boostorg/boost.git"
-export ZOPEN_DEV_DEPS="cmake make zoslib which coreutils comp_clang sed jq"
+export ZOPEN_DEV_DEPS="cmake make zoslib which coreutils comp_clang sed jq git findutils diffutils bzip2 zlib"
 export ZOPEN_BUILD_LINE="DEV"
 export ZOPEN_CLONE_SUBMODULES="yes"
 export ZOPEN_COMP="clang"
-export ZOPEN_CXXFLAGS ="-m64"
 
-export ZOPEN_BOOTSTRAP="./bootstrap.sh"
+export ZOPEN_BOOTSTRAP="zopen_bootstrap"
 export ZOPEN_BOOTSTRAP_OPTS="-with-libraries=thread,chrono --with-toolset=clang --prefix=$ZOPEN_INSTALL_DIR"
 export ZOPEN_CONFIGURE=skip
+export ZOPEN_CHECK="zopen_check"
 export ZOPEN_CLEAN=skip
-export ZOPEN_MAKE="./b2"
+export ZOPEN_MAKE="zopen_build"
 export ZOPEN_MAKE_MINIMAL="yes"
 export GIT_DISCOVERY_ACROSS_FILESYSTEM="yes"
+export ZOPEN_INSTALL="zopen_install"
 
+zopen_bootstrap()
+{
+        echo "In bootstrap"
```

Comment:
Can we remove these?

Reply:
Done, retained few for debugging. Thank you!